src/anm/animation/element.js

```diff
@@ -2884,6 +2885,46 @@ Element.getMatrixOf = function(elm, m) {
     return t;
 };
 
+Element.getTranslate = function(elm) {
+    if (elm.parent.layer2Bone) {
+	elm.$bonePath = elm.$bonePath || Element.bonePath(elm);
+        var result = {x: 0; y: 0};
+        var bone = null;
+	for (var li = elm.$bonePath.length; li--;) {
+            bone = elm.$bonePath[li];
+	    result.x += bone.bonelength * Math.cos(bone.bonerotate);
+	    result.y += bone.bonelength * Math.sin(bone.bonerotate); 
+	}
+        elm = bone ? elm.parent.children[bone.$from] : elm;
+        result.x += elm.x;
```

Comment:
I think if this tween will be assigned to the every child and this child will check its parent, this could work. But `elm.x` will be taken from a previous frame, is that ok?